pyproject.toml

```diff
@@ -63,6 +108,15 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[tool.uv.workspace]
+members = ["tracing", "testing"]
+exclude = ["packages/seeds"]
```

Comment:
What's this? A web search didn't find anything obvious and it doesn't seem to be something in the repo. Could we add a comment explaining it?